pkgs/development/python-modules/rctclient/default.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  #  aiohttp,
+  buildPythonPackage,
+  fetchFromGitHub,
+  #pytest-aiohttp,
+  #pytestCheckHook,
+  pythonOlder,
+  typing,
+  click,
+  scapy,
+  sphinx,
+  sphinx-autodoc-typehints,
+  sphinx-rtd-theme,
+}:
+
+buildPythonPackage rec {
+  pname = "rctclient";
+  version = "0.0.4";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "svalouch";
+    repo = "python-rctclient";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-QPla5h8wSM9Ynj44Uwc1a2yAnu8TXbyBWzVHQeW6jnI=";
+
+  };
+
+  postPatch = ''
+    # see comment on https://github.com/cmroche/moonraker-api/commit/e5ca8ab60d2839e150a81182fbe65255d84b4e4e
+    #substituteInPlace setup.py \
+    #  --replace 'name="rctclient",' 'name="rctclient",version="${version}",'
+  '';
+
+  propagatedBuildInputs = [
+    typing
+    click
+    scapy
+    sphinx
+    sphinx-autodoc-typehints
+    sphinx-rtd-theme
+  ];
+
+  nativeCheckInputs = [
+    #  pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "rctclient" ];
+
+  meta = with lib; {
```

```suggestion
  meta = with lib; {
    changelog = "..."
```

Add a changelog URL pattern.

---

Thanks for your review and sorry for the many mistakes. I had prepared this a while back and thought I had cleaned it up, but that was not the case... m(
Was successfully tested again with the applied changes on the hardware today and I hope this is good to go now.